### DISCUSSION TITLE
TypeError: cross_entropy_loss(): argument 'target' (position 2) must be Tensor, not tuple

### DISCUSSION THREAD

**Discussion by DanceElephant** [rasbt/comparing-automatic-augmentation-blog#2] [General]
I used my own local data set, and have stored the data set information in the dictionary like cifar10, running the error: TypeError: cross_entropy_loss(): argument 'target' (position 2) must be Tensor, not tuple
The error location appears in torch\nn\functional.py
Could you help me with this question? Thank you very much

**Comment by rasbt** [comment]
Hey there,

it sounds like you are perhaps passing both, the features and the targets to the cross entropy loss.
